libs/cohere/langchain_cohere/rerank.py

```diff
@@ -31,14 +31,14 @@ class Config:
         extra = Extra.forbid
         arbitrary_types_allowed = True
 
-    @root_validator(pre=True)
+    @root_validator()
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key and python package exists in environment."""
         if not values.get("client"):
             cohere_api_key = get_from_dict_or_env(
                 values, "cohere_api_key", "COHERE_API_KEY"
             )
-            client_name = values.get("user_agent", "langchain")
+            client_name = values.get("user_agent")
```

Comment:
why dropping the default?

Reply:
good point - the other implementations ([e.g.](https://github.com/langchain-ai/langchain-cohere/blob/main/libs/cohere/langchain_cohere/llms.py#L84C31-L84C41)) don't have a default because it's guaranteed the class field is there. I can add it back if you feel strongly though!

Reply:
if it's guaranteed, let's use `values["user_agent"]` instead?

Reply:
sure!